### DISCUSSION TITLE
HookWebpackError from tailwind

### DISCUSSION THREAD

**Discussion by Cedar-81** [tailwindlabs/tailwindcss#10794] [Help] [answered]
I have been using tailwind successfully with Nextjs 13.2.4 and recently I tried to dynamically update the background image property of a div using tailwind and I suddenly ran into this error
![IMG_20230314_204617.jpg](https://user-images.githubusercontent.com/64336732/225120751-4811bc95-28ff-4ce8-85ef-eba6c806af8c.jpg)

At first I had thought it was as a result of what I had done so I commented out the code and tried again but still got the same error after hours of debugging I found out the only way to get rid of the error is by commenting out the @tailwind utilities rule in my global.css or uninstall tailwind completely.

Tbh, I really really appreciate what you guys have done with tailwind and would be extremely grateful if a solution to this is presented or a work around at the very least. Here are some images for more information.

![IMG_20230314_204627.jpg](https://user-images.githubusercontent.com/64336732/225121511-bd199077-b250-4d9e-a838-e3b82399fa07.jpg)
![IMG_20230314_204641.jpg](https://user-images.githubusercontent.com/64336732/225121617-25fc6ec8-8900-418b-880b-0d4b9b66adc0.jpg)
![IMG_20230314_204712.jpg](https://user-images.githubusercontent.com/64336732/225121666-f24908f1-c708-47cd-a61f-7ba73c87f1d6.jpg)

Pls if any other info is needed let me know. Thanks slot.

**Comment by wongjn** [comment, marked as answer]
From the error message, it seems like you have tried to use string interpolation to build a class name, like `bg-[url(${url})]`, but this will not work, see https://tailwindcss.com/docs/content-configuration#dynamic-class-names. Instead consider applying the `background-image` value via the `style` attribute.

**Reply by Cedar-81** [reply]
Hmm... Thanks, that makes sense, the problem now is that I am still getting this error even after completely commenting out the component implementing that feature, I even commented out my entire page but I am still having this error for some reason. I had thought it was some kind of caching but I have unistalled and installed tailwind and also deleted my .next folder still to no avail.

**Reply by wongjn** [reply]
The string would still exist in the file content that Tailwind scans for so it would still produce the errant rule and thus you would still receive the error. Remove the string completely instead of commenting it out and the error should not appear anymore.

**Reply by Cedar-81** [reply]
Oh, that's makes a lot of sense, would try that out and get back. Thanks a lot, I really appreciate ur help.

**Reply by Cedar-81** [reply]
this fixed my issue thanks a lot. I'd be closing this issue now. I'm very grateful.
